functions/export-csv.js

```diff
@@ -124,4 +129,11 @@ function getMultipleChoiceValues(id, element) {
   return option.code || option.label["en"] || "";
 }
 
+/**
+ * Returns the file name in lowercase (replacing any special characters with '-') for csv export
+ */
+function getFileName(layerName) {
+  return layerName.replace(/[^a-z0-9]/gi, "-").toLowerCase() + ".csv";
```

Comment:
It appears you're converting to lower case _after_ slugifying, which will replace all uppercase characters with `-`.

Also, we should probably specify a default in case layerName is empty. How about:
```js
layerName = layerName || 'ground-export';
fileBase = layerName.toLowerCase().replace(/[^a-z0-9]/gi, "-");
return `${layerName}.csv`;
```

Reply:
> It appears you're converting to lower case _after_ slugifying, which will replace all uppercase characters with `-`.
> 
> Also, we should probably specify a default in case layerName is empty. How about:
> 
> ```js
> layerName = layerName || 'ground-export';
> fileBase = layerName.toLowerCase().replace(/[^a-z0-9]/gi, "-");
> return `${layerName}.csv`;
> ```

Example seem to work fine as it will first convert everything to lowercase and then replace special chars with '-':

<img width="800" alt="Screen Shot 2021-03-31 at 11 21 54 PM" src="https://user-images.githubusercontent.com/45903885/113243436-a1032200-9278-11eb-9228-8d4a98090bb5.png">